bin/katello-certs-check

```diff
@@ -182,6 +182,18 @@ function check-ca-bundle-size () {
     fi
 }
 
+function check-ca-bundle-trust-rules () {
+    printf "Checking if CA bundle has trust rules: "
+    CHECK=$(awk -v cmd='openssl x509 -noout -text' ' /BEGIN/{close(cmd)};{print | cmd}'  $CA_BUNDLE_FILE |grep 'Trusted Uses'|wc -l)
+    printf $CHECK
+    if [[ $CHECK -lt 1 ]]; then
+		success
+    else
+		error 10 "The CA bundle contains $CHECK certificate(s) with trust rules. This may create problems for older systems to trust the bundle. Please, recreate the bundle using certificates without trust rules"
```

Comment:
```suggestion
	error 10 "The CA bundle contains $CHECK certificate(s) with trust rules. This may create problems for older systems to trust the bundle. Please, recreate the bundle using certificates without trust rules"
```

Reply:
ack